Sources/CorePayments/Networking/GraphQL/GraphQLClient.swift

```diff
@@ -1,17 +1,17 @@
 import Foundation
 
-class GraphQLClient {
+public class GraphQLClient {
```

Comment:
```suggestion
/// :nodoc:
public class GraphQLClient {
```

If public but meant to be hidden from merchants, we have to no-doc it.

Reply:
Addressed in 8a67bce871356c3a8bd910be35cc7efd64d1864c